src/olympia/addons/serializers.py

```diff
@@ -295,19 +299,24 @@ class ESBaseAddonSerializer(BaseESSerializer):
     translated_fields = ('name', 'description', 'homepage', 'summary',
                          'support_email', 'support_url')
 
+    def fake_file_object(self, obj, data):
+        file_ = File(
+            id=data['id'], created=self.handle_date(data['created']),
+            hash=data['hash'], filename=data['filename'],
+            platform=data['platform'], size=data['size'],
+            status=data['status'], version=obj)
+        file_.webext_permissions_list = data['webext_permissions_list']
```

Comment:
Use `data.get('webext_permissions_list', [])` in order not to break serialization with the old ES index that doesn't have that key yet.